### DISCUSSION TITLE
Is there any way to get data like shot vectors/impacts?

### DISCUSSION THREAD

**Discussion by Theoya** [roflmuffin/CounterStrikeSharp#582] [Q&A] [answered]
Is there any way to get data like shot vectors/impacts and aim direction? Trying to build a practice tool and I want to track where the player is shooting

**Comment by KillStr3aK** [comment, marked as answer]
> Is there any way to get data like shot vectors/impacts and aim direction? Trying to build a practice tool and I want to track where the player is shooting

Hi, you can use the [EventBulletImpact](https://docs.cssharp.dev/api/CounterStrikeSharp.API.Core.EventBulletImpact.html) event
